meson.build

```diff
@@ -280,7 +280,7 @@ libcurl = dependency('libcurl', version: '>= 7.62.0', required: get_option('curl
 if libcurl.found()
   conf.set('HAVE_LIBCURL', '1')
 endif
-libdrm = dependency('libdrm', required: get_option('libdrm'))
+libdrm = dependency('libdrm', required: false)
```

Comment:
can drop from summary I would say now too

Reply:
I think the optional ones still make sense -- but I also noticed curl (now a hard dep) there so I remove that instead.